FinnUI.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'FinnUI'
-  s.version      = '20.0.0'
+  s.version      = '21.0.0'
```

Comment:
Please use the fastlane release script to make new versions after the PR is merged instead 🙏